src/vs-bicep/Bicep.VSLanguageServerClient.TestServices/LanguageServerActivationService.cs

```diff
@@ -26,35 +24,36 @@ public void WaitForLanguageServerActivation()
                 var serviceForComponentModel = ServiceProvider.GlobalProvider.GetService(typeof(SComponentModel).GUID);
                 if (serviceForComponentModel is IComponentModel componentModel)
                 {
-                    var languageServiceBrokerExports = componentModel.DefaultExportProvider.GetExports<ILanguageServiceBroker2>();
+                    // Instead of using this obsolete interface, either look for semantic tokens in our output log or see if there's
+                    //   another way to tell lif a language server is active.
+                    // Tracked here: https://github.com/Azure/bicep/issues/8078
 
-                    if (languageServiceBrokerExports is null || !languageServiceBrokerExports.Any())
-                    {
-                        throw new Exception("Did not find any exports for ILanguageServiceBroker2");
-                    }
-
-                    var languageServiceBroker = languageServiceBrokerExports.First().Value;
+                    //var languageServiceBrokerExports = componentModel.DefaultExportProvider.GetExports<ILanguageServiceBroker2>();
```

Comment:
If we are going to verify language server activation via semantic tokens, should we just delete this instead of commenting?

Reply:
Same response :-)